Svc/Health/test/ut/Tester.cpp

```diff
@@ -14,8 +14,11 @@
 #include <Fw/Test/UnitTest.hpp>
 
 #define INSTANCE 0
-#define MAX_HISTORY_SIZE 2000
+#define MAX_HISTORY_SIZE (Svc::HealthComponentBase::NUM_PINGSEND_OUTPUT_PORTS * Svc::HealthComponentBase::NUM_PINGSEND_OUTPUT_PORTS * Svc::HealthComponentBase::NUM_PINGSEND_OUTPUT_PORTS)
```

Comment:
Nit pick: `Svc::HealthComponentBase::NUM_PINGSEND_OUTPUT_PORTS * 3`

Reply:
This is actually cubic in nature, not linear.